src/deploydocs.jl

```diff
@@ -289,35 +295,37 @@ function deploydocs(;
             mktempdir() do temp
                 git_push(
                     root, temp, deploy_repo;
-                    branch=deploy_branch, dirname=dirname, target=target,
-                    sha=sha, deploy_config=deploy_config, subfolder=deploy_subfolder,
-                    cname=cname, devurl=devurl,
-                    versions=versions, forcepush=forcepush,
-                    is_preview=deploy_is_preview, archive=archive,
+                    branch = deploy_branch, dirname = dirname, target = target,
+                    sha = sha, deploy_config = deploy_config, subfolder = deploy_subfolder,
+                    cname = cname, devurl = devurl,
+                    versions = versions, forcepush = forcepush,
+                    is_preview = deploy_is_preview, archive = archive,
                 )
             end
         end
     end
+    return
 end
 
 
 function _get_inventory_version(objects_inv)
-    open(objects_inv) do input
+    v = open(objects_inv) do input
         for line in eachline(input)
             if startswith(line, "# Version:")
                 return strip(line[11:end])
             end
         end
         error("Invalid $objects_inv: missing or invalid version line")
     end
+    return v
```

Comment:
Did the formatter do this? :open_mouth: I would the automatic change to be just a `return` in front of `open`.